From 20aed2ec8690f23d09fc8eb2c3c7a0e8f32e74f3 Mon Sep 17 00:00:00 2001
From: Stefan Assmann <sassmann@redhat.com>
Date: Tue, 26 Aug 2014 14:11:47 -0400
Subject: [ethernet] igb: Fix code comment

Message-id: <1409062359-13063-30-git-send-email-sassmann@redhat.com>
Patchwork-id: 89177
O-Subject: [RHEL7.1 PATCH 29/81] igb: Fix code comment
Bugzilla: 1091121
RH-Acked-by: Nikolay Aleksandrov <nikolay@redhat.com>
RH-Acked-by: Neil Horman <nhorman@redhat.com>
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>

From: Jeff Kirsher <jeffrey.t.kirsher@intel.com>

Recently added code comment was missing a space that is needed.

Signed-off-by: Jeff Kirsher <jeffrey.t.kirsher@intel.com>
Signed-off-by: Stefan Assmann <sassmann@redhat.com>
(cherry picked from commit b936136da2223be28452162c3dd22c664a8c7f16)
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/drivers/net/ethernet/intel/igb/igb_main.c b/drivers/net/ethernet/intel/igb/igb_main.c
index 340a344..ea8b9c4 100644
--- a/drivers/net/ethernet/intel/igb/igb_main.c
+++ b/drivers/net/ethernet/intel/igb/igb_main.c
@@ -1978,7 +1978,7 @@ void igb_reset(struct igb_adapter *adapter)
   }
  }
 #endif
- /*Re-establish EEE setting */
+ /* Re-establish EEE setting */
  if (hw->phy.media_type == e1000_media_type_copper) {
   switch (mac->type) {
   case e1000_i350:
-- 
1.7.1